File: 03.JAVASCRIPT/pg-166-169/pg_166_09.js
console.log("");
console.log("JSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJS");
console.log("READ A TWO-DIGIT INTEGER AND DETERMINE IF ONE OF ITS DIGITS IS A MULTIPLE OF THE OTHER");
console.log("JSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJSJS");
console.log("");

let numInput = document.querySelector(".num");
let btnProof = document.querySelector(".btnProof");
let textar = document.querySelector(".textar");
let dig1, dig2;

function validateNumber()
{
    let num = Number(numInput.value);

    if(num < 0)
        num *= -1;

    if(num >= 10 && num <= 99)
    {
        dig2 = num % 10;
        dig1 = Math.floor(num / 10);

        if(dig1 % dig2 == 0)
        {
            textar.value = "The first digit is multiple of the second digit.";
        }
        else
        {
            textar.value = "The first digit isn't multiple of the second digit.";
        }

        if(dig2 % dig1 == 0)
        {
            textar.value += "The second digit is multiple of the first digit.";
        }
        else
        {
            textar.value += "The second digit isn't multiple of the first digit.";
        }
    }
    else
    {
        textar.value = "The written number doesn't have two digits. Please try again!";
    }
}


btnProof.addEventListener("click", validateNumber);
numInput.addEventListener("keydown", function(event)
{
    if(event.key === "Enter")
    {
        validateNumber();
    }
});

File: 01.JAVA/pg-166-169/pg_166_09.java
import java.util.Scanner;

public class pg_166_09
{
    public static void main(String[]args)
    {
        Scanner enter = new Scanner(System.in);
        int num, dig1, dig2;

        System.out.println("");
        System.out.println("JVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJV");
        System.out.println("READ A TWO-DIGIT INTEGER AND DETERMINE IF ONE OF ITS DIGITS IS A MULTIPLE OF THE OTHER");
        System.out.println("JVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJV");
        System.out.println("");

        System.out.print("Write a two-digit integer: ");
        num = enter.nextInt();

        if(num < 0)
        {
            num *= -1;
        }

        if(num >= 10 && num <= 99)
        {
            dig2 = num % 10;
            dig1 = num / 10;

            if(dig1 % dig2 == 0)
            {
                System.out.println("The frst digit is multiple of the second digit.");
            }
            else
            {
                System.out.println("The first digit isn't multiple of the second digit.");
            }
            if(dig2 % dig1 == 0)
            {
                System.out.println("The second digt is multiple of the first digit.");
            }
            else
            {
                System.out.println("The second digit isn't multiple of the first digit.");
            }
        }
        else
        {
            System.out.println("The written number doesn't have two digits. Please try again!");
        }

    }
}